### PR TITLE
Minor changes on create-jobs

### PR DESCRIPTION
**Pull request by loysollivier**
Minor changes on help and required parameters for create-jobs

### CODE REVIEW

**Review by jbrun3t — COMMENTED** [review]
Cool with me..

I'm curious, is it some kind of python convention/rule ?

**Review by jbrun3t — CHANGES_REQUESTED** [review]
As we discussed, boot should be required but no default provided. Right now only ramdisk works for qemu and nbd for the rest

File: utils/create-jobs.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def parse_cmdline(machines, tests, rfs_types):
                         help="url fetch base",
                         default='https://download.automotivelinux.org/AGL/upload/ci/')
     parser.add_argument('--boot', action='store', dest='rfs_type', nargs = 1,
-                        choices=rfs_types, help='select boot type')
+                        choices=rfs_types, help='select boot type', default='ramdisk')
```
Comment (jbrun3t) [inline utils/create-jobs.py]: As we discussed, boot should be required but no default provided. Right now only ramdisk works for qemu and nbd for the rest

File: utils/create-jobs.py [hunk 2]
```diff
@@ -6,7 +6,8 @@
 import urlparse
 
 def parse_cmdline(machines, tests, rfs_types):
-    parser = argparse.ArgumentParser(description="AGL create job")
+    parser = argparse.ArgumentParser(description="AGL create job",
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```
Comment (jbrun3t) [inline utils/create-jobs.py]: Ack

**Comment by loysollivier** [comment]
Yes: https://www.python.org/dev/peps/pep-0008/
It is used in kernelCI as well. Not mandatory but I like the readability and I'm used to it now

**Review by jbrun3t — APPROVED** [review]

File: utils/create-jobs.py (hunk 1 above)
Comment (loysollivier) [reply utils/create-jobs.py]: This was removed in a later commit after discussing with you

File: utils/create-jobs.py [hunk 3]
```diff
@@ -44,7 +44,7 @@ def main():
         args.job_name += ' - {}'.format(args.job_id)
 
         if args.job_index is not None:
-            args.urlbase = urlparse.urljoin(args.urlbase, args.job_index + '/')
+            args.urlbase = urlparse.urljoin(args.urlbase, args.job_index)
```
Comment (jbrun3t) [inline utils/create-jobs.py]: Are you sure about this ? I remember I had issue with urljoin when the trailing '/' was missing, it discarded the last directory

**Review by jbrun3t — APPROVED** [review]

File: utils/create-jobs.py (hunk 3 above)
Comment (loysollivier) [reply utils/create-jobs.py]: Yes, otherwise you'll get double / in the url but I had the same issue for the first one

**Comment by jbrun3t** [comment]
Sorry, not really used to code review on git hub yet ... Series Looks good overall. Please squash patch 2 and 7

**Comment by khilman** [comment]
Merged, after squashing patches 2 & 7 manually.

**Pull request closed without merging**